### ISSUE TITLE
Meaning of phenotype terms

### ISSUE THREAD

**Issue by dbrnz** [open-physiology/apinatomy-models#36]
We still have:
```
ilxtr:neuron-type-sstom-11 phenotype ['ILX:0105486'] is unknown
ilxtr:neuron-type-sstom-12 phenotype ['ILX:0104003'] is unknown
ilxtr:neuron-type-sstom-13 phenotype ['ILX:0105486'] is unknown
ilxtr:neuron-type-sstom-14 phenotype ['ILX:0104003'] is unknown
```

What is the meaning of these terms?

@tgbugs commented about this [here](https://github.com/open-physiology/apinatomy-models/issues/29#issuecomment-1275401387) but these terms still appear...

**Comment by tgbugs** [comment]
Here is the mapping.

```python
IlxTerm('ILX:0104003', label='Excitatory')
IlxTerm('ILX:0105486', label='Inhibitory')
```

**Comment by dbrnz** [comment]
Thanks.

The other phenotype terms though are like `ilxtr:SensoryPhenotype`, in the `ilxtr:` namespace. Should these ILX ones not be `ilxtr:Excitatory` and `ilxtr:Inhibitory`?

**Comment by tgbugs** [comment]
I have never used them as such, because I used the `ILX:` form directly in the python code since it already existed (I usually only use `ilxtr:` in cases where a term does not exist yet or when I don't know whether the semantics I need in a particular context match the modelling in some other context).

**Comment by dbrnz** [comment]
So are you saying that the `ilxtr:SensoryPhenotype` and similar `ilxtr:` terms that SCKAN uses for phenotypes will eventually become `ILX:` terms??

**Comment by tgbugs** [comment]
Yes, they will eventually become `ILX:` terms, however we will not switch the identifiers in the ontology so no changes would need to be made on your end. When the receive a corresponding `ILX:` term they will redirect to that term.

**Comment by dbrnz** [comment]
At the moment we get this mixture of `ilxtr:` and `ILX:` terms for phenotypes -- does it not make sense for them to all be in the same namespace? (I know from a RDF point-of-view it makes no difference; just me wanting consistency!)

**Comment by tgbugs** [comment]
The use cases for the various prefixes are determined by requirements that are mostly orthogonal to this, precisely because from an RDF standpoint it makes no difference, and if we started trying to clean up and make everything consistent and human readable/accessible we could spend all our time on that :).

**Comment by dbrnz** [comment]
As an aside, parts of the codebase rely on namespace prefixes having a consistent meaning, which is not assured by RDF. Hopefully this won't trip anyone up, but things are likely to stop working if different prefix is used for a namespace...
